client/src/assets/search.svg

```diff
@@ -0,0 +1,4 @@
+<!-- from Bootstrap Icons (https://icons.getbootstrap.com/icons/search/) -->
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
```

Comment:
Use the bootstrap icons npm module

Reply:
btw we don't have bootstrap icons installed yet, and it's coming in this PR (#88)
going to wait for 88 to land before making the change

Reply:
#88 merged!